velox/exec/WindowFunction.h

```diff
@@ -71,7 +71,8 @@ class WindowFunction {
       const BufferPtr& frameStarts,
       const BufferPtr& frameEnds,
       vector_size_t resultOffset,
-      const VectorPtr& result) = 0;
+      const VectorPtr& result,
+      const vector_size_t numPartitionRows) = 0;
```

Comment:
Why do you need to make this change ? You can get this from the WindowPartition directly righ t ? The WindowPartition can be saved during the resetPartition call.

Reply:
Oh right, thanks for pointing it out, reverted this change.